src/components/Search/SearchPageHeader.tsx

```diff
@@ -131,11 +131,23 @@ function SearchPageHeader({query, selectedItems = {}, hash, clearSelectedItems}:
             icon={headerContent[query]?.icon}
             shouldShowBackButton={false}
         >
-            {getHeaderButtons()}
+            {headerButtonsOptions.length > 0 && (
+                <ButtonWithDropdownMenu
+                    onPress={() => null}
```

Comment:
is that correct? that is just the action on pressing the main button that would display the list, and each option has their own onPress?
Or are we not yet ready to call the actual action?

Reply:
onPress function fires only when we pass `isSplitButton` prop - in this case we don't need it